chicken-sloths/bangazon-api-sprint1
# build data

# Description
Wrote `data/build-data.js`, which generates JSON files in `/data/json`:
- `computers.json`
- `customers.json`
- `departments.json`
- `employeeComputers.json`
- `employees.json`
- `employeeTrainingPrograms.json`
- `orders.json`
- `paymentOptions.json`
- `products.json`
- `productOrders.json`
- `productTypes.json`
- `trainingPrograms.json`

## Related Ticket(s)
fixes #18, fixes #39, fixes #37 

## Problem to Solve
`npm run data:generate` runs now

## Steps to Test Solution
- delete `data/json`
- run `npm run data:generate`
- you should see those files reappear
- _refresh your file tree in VSCode_
